frontend/rolecall/src/app/user/user-editor.component.ts

```diff
@@ -291,42 +296,44 @@ export class UserEditor implements OnInit {
     if (info.type == "date") {
       let date = Date.parse(val.value);
       val = date;
-      this.userSaved = false;
+      this.disableSave = false;
     }
     else if (info.type == "permissions") {
       let val2 = this.workingUser.has_permissions;
       for (let entry of Object.entries(val2)) {
         if (val.includes(entry[0])) {
           if (!val2[entry[0]]) {
-            this.userSaved = false;
+            this.disableSave = false;
           }
           val2[entry[0]] = true;
         } else {
           if (val2[entry[0]]) {
-            this.userSaved = false;
+            this.disableSave = false;
           }
           val2[entry[0]] = false;
         }
       }
       val = val2;
-    }
-    else if (key == "Privilege Classes") {
-      if (this.workingUser.has_privilege_classes) {
-        let largerArr = val.length > this.workingUser.has_privilege_classes.length ? val : this.workingUser.has_privilege_classes;
-        let smallerArr = val.length <= this.workingUser.has_privilege_classes.length ? val : this.workingUser.has_privilege_classes;
-        for (let i = 0; i < largerArr.length; i++) {
-          if (largerArr[i] != smallerArr[i]) {
-            this.userSaved = false;
-            break;
+    } else if (info.type == "roles") {
+      let val2 = this.workingUser.has_roles;
+      for (let entry of Object.entries(val2)) {
```

Comment:
Rather than `entry` let's use a var name that describes what the entry is.

Reply:
Yes. But see comment above.